src/Symfony/Component/HttpFoundation/Tests/InputBagTest.php

```diff
@@ -36,6 +39,58 @@ public function __toString(): string
         $this->assertFalse($bag->get('bool'), '->get() gets the value of a bool parameter');
     }
 
+    /**
+     * @group legacy
+     */
+    public function testGetIntError()
+    {
+        $this->expectDeprecation('Since symfony/http-foundation 6.3: Not passing flag "FILTER_NULL_ON_FAILURE" to "Symfony\Component\HttpFoundation\InputBag::filter(foo)" is deprecated. In Symfony 7.0, this method will throw a "Symfony\Component\HttpFoundation\Exception\BadRequestException" when the value is invalid and this flag is omitted.');
```

Comment:
I have one final concern about the deprecation messages: they don't include the context of the called method (`getInt` or `getBoolean`), and the proposed solution of adding a flag isn't applicable to these methods.